.stylelintrc.json

```diff
@@ -4,9 +4,10 @@
 		"at-rule-no-unknown": [
 			true,
 			{
-				"ignoreAtRules": [ "define-mixin", "mixin" ]
+				"ignoreAtRules": [ "define-mixin", "mixin", "svg-load" ]
 			}
 		],
-		"selector-class-pattern": null
+		"selector-class-pattern": null,
```

Comment:
Curious what this rule ignore solved?

Reply:
That one was pre-existing, the line changed came from the comma needed for the next rule